bench/wtperf/wtperf_opt.i

```diff
@@ -157,49 +147,43 @@ DEF_OPT_AS_UINT32(run_ops, 0, "total insert, modify, read and update workload op
 DEF_OPT_AS_UINT32(run_time, 0, "total workload seconds")
 DEF_OPT_AS_UINT32(sample_interval, 0, "performance logging every interval seconds, 0 to disable")
 DEF_OPT_AS_UINT32(sample_rate, 50,
-  "how often the latency of operations is measured. One for every operation,"
-  "two for every second operation, three for every third operation etc.")
+  "how often the latency of operations is measured. One for every operation,two for every second "
```

Comment:
There's a space missing after the comma (although it looks like it was missing in the original as well).

Reply:
83e4aa4